python/pyspark/ml/tests.py

```diff
@@ -1980,8 +1997,8 @@ def test_java_params(self):
                    pyspark.ml.regression]
         for module in modules:
             for name, cls in inspect.getmembers(module, inspect.isclass):
-                if not name.endswith('Model') and issubclass(cls, JavaParams)\
-                        and not inspect.isabstract(cls):
+                if not name.endswith('Model') and not name.endswith('Params')\
```

Comment:
Just to make sure I've understood whats happening here, were avoiding doing the default params test on non-concrete classes like the base params shared between the model and the estimator and instead testing just the model and estimator on their own right?

Reply:
Yes, that's pretty much right but this is only checking estimators and skips models also.  We should have an explicit check for `CountVectorizer.from_vocabulary` here too since that is possible.  Unfortunately, a new param `maxDF` was added to Scala recently and the param check will fail.  Once that is in Python, we can add the check for it here.

Reply:
Sounds reasonable. I look forward to us automatically catching models with missing params eventually as well.

Reply:
I'm helping get `maxDF` in python now, so after that's done I'll make a followup to add this